runtime/parachains/src/inclusion.rs

```diff
@@ -189,33 +193,42 @@ decl_error! {
 		/// The validation code hash of the candidate is not valid.
 		InvalidValidationCodeHash,
 	}
-}
-
-decl_event! {
-	pub enum Event<T> where <T as frame_system::Config>::Hash {
-		/// A candidate was backed. `[candidate, head_data]`
-		CandidateBacked(CandidateReceipt<Hash>, HeadData, CoreIndex, GroupIndex),
-		/// A candidate was included. `[candidate, head_data]`
-		CandidateIncluded(CandidateReceipt<Hash>, HeadData, CoreIndex, GroupIndex),
-		/// A candidate timed out. `[candidate, head_data]`
-		CandidateTimedOut(CandidateReceipt<Hash>, HeadData, CoreIndex),
-	}
-}
 
-decl_module! {
-	/// The parachain-candidate inclusion module.
-	pub struct Module<T: Config>
-		for enum Call where origin: <T as frame_system::Config>::Origin
-	{
-		type Error = Error<T>;
+	/// The latest bitfield for each validator, referred to by their index in the validator set.
+	#[pallet::storage]
+	pub type AvailabilityBitfields<T: Config> = StorageMap<
```

Comment:
Wrong visibility on each of the storage items, this and the other ones below:

```suggestion
	pub(crate) type AvailabilityBitfields<T: Config> = StorageMap<
```

Reply:
yeah, I think the other pallets I migrated don't need public facing storage either